RELEASE/scripts/autoscend/iotms/mr2024.ash

```diff
@@ -721,6 +730,12 @@ boolean auto_getClanPhotoBoothEffect(string ef_string, int n_times)
 		return false;
 	}
 	
+	n_times = min(n_times,auto_remainingClanPhotoBoothEffects());
+	if (n_times <= 1)
```

Comment:
<1 or = 0, right? If it's exactly 1 you want to get it once.

Reply:
Correct.